### ISSUE TITLE
When step function doesn't exist, code hangs.

### ISSUE THREAD

**Issue by sam-n-johnston** [jefer590/serverless-offline-step-functions#85]
**Describe the bug**
When step function doesn't exist, code hangs.

**To Reproduce**
1. Create a new step func:
```
stepFunctions:
    stateMachines:
        MyStep: 
            definition:
                ...
```
2. Make a lambda call `stepFunction.execute`, but with `MyStep1` as the step function.
3. Notice that it hangs at `[Step Functions API Simulator] Got request for POST /` forever 

**Expected behavior**
It should log an error and continue

**Comment by sam-n-johnston** [comment]
You can assign me to this

**Comment by sam-n-johnston** [comment]
Resolved in PR #104
